packages/app/src/components/home/HomePage.tsx

```diff
@@ -46,7 +49,13 @@ export const HomePage = () => {
             />
           </Grid>
           <Grid item xs={12} md={6}>
-            <CloudsmithStatsCard repo="roadie-npm" owner="roadie" />
+            <CloudsmithStatsCard
+              repo="bartosz-testing"
```

Comment:
I think we should remove this card or use the roadie-repo one instead of your name.